README.md

```diff
@@ -44,41 +44,79 @@ Get started with the benchmarks:
 
 ## Benchmark Results
 
-<!-- PERFORMANCE_RESULTS_START -->
+```graphql
+{
+  posts {
+    title
+  }
+}
+```
+
+<!-- PERFORMANCE_RESULTS_START_1 -->
 
 | Server | Requests/sec | Latency (ms) |
 |--------:|--------------:|--------------:|
-| [Caliban] | `9,355.17` | `11.02` |
-| [async-graphql] | `8,004.78` | `12.49` |
-| [Tailcall] | `7,424.98` | `13.45` |
-| [Gqlgen] | `2,208.32` | `46.65` |
-| [Apollo GraphQL] | `1,845.58` | `53.99` |
-| [Netflix DGS] | `1,666.14` | `63.75` |
+| [Tailcall] | `56,655.60` | `1.75` |
+| [Caliban] | `8,833.37` | `11.79` |
+| [async-graphql] | `7,027.24` | `14.34` |
+| [Gqlgen] | `1,969.06` | `51.76` |
+| [Apollo GraphQL] | `1,724.77` | `57.78` |
+| [Netflix DGS] | `1,507.36` | `70.39` |
```

Comment:
Results need to be formatted as follows:


| Query | Server                             | Requests/sec | Latency (ms) |
| ----- | ---------------------------------- | -----------: | -----------: |
| 1.    | `{posts {title body user {name}}}` |              |
|       | Tailcall                           |    28,389.70 |         3.52 |
|       | async-graphql                      |     2,411.80 |        41.48 |
|       | Caliban                            |     1,416.62 |        70.59 |
|       | Gqlgen                             |     1,343.30 |        77.00 |
|       | Apollo GraphQL                     |       791.84 |       126.09 |
|       | Netflix DGS                        |       340.42 |       219.50 |
| 2.    | `{posts {title body}}`             |
|       | Tailcall                           |    56,655.60 |         1.75 |
|       | Caliban                            |     8,833.37 |        11.79 |
|       | async-graphql                      |     7,027.24 |        14.34 |
|       | Gqlgen                             |     1,969.06 |        51.76 |
|       | Apollo GraphQL                     |     1,724.77 |        57.78 |
|       | Netflix DGS                        |     1,507.36 |        70.39 |

Reply:
The current formatting is too difficult to read. I recommend the above format so that we can keep adding more tests easily. I would also add a column which can give a comparative performance WRT Tailcall, so for eg: Tailcall is 100%, then other could be 10%, 50% etc.